.gitignore

```diff
@@ -1,35 +1,37 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I need to go through here and figure out what we need to change and keep and update.